main.py

```diff
@@ -14,13 +14,14 @@ def reEmail(fname):
   
   fhand = open(fname,'r')
   for line in fhand:
-    extr = re.findall('.*',line)
+    extr = re.findall('\S*@\S*',line)
```

Comment:
This will allow "@Mr.Pauls" because the * symbol is 0 or more. 

`extr = re.findall('\S+@\S+',line)`

Reply:
The COUNT is missing.